YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java

```diff
@@ -272,10 +276,48 @@ private void showVoiceOption() {
     }
 
     @Override
-    public void onResume() {
-        super.onResume();
+    protected void onStart() {
+        super.onStart();
+        fh = new WebviewOverlay();
+        FragmentManager fragManager = getSupportFragmentManager();
+        fragManager.beginTransaction()
+                .add(R.id.container, fh)
+                .commit();
+    }
+
+    @Override
+    protected void onStop() {
+        UpdateAgentStatus("offline");
```

Comment:
method name should start from small letter

Reply:
sure